control/lti.py

```diff
@@ -111,11 +111,56 @@ def damp(self):
         Z = -real(splane_poles)/wn
         return wn, Z, poles
 
+    def frequency_response(self, omega, squeeze=True):
+        """Evaluate the linear time-invariant system at an array of angular
+        frequencies.
+
+        Reports the frequency response of the system,
+
+             G(j*omega) = mag*exp(j*phase)
+
+        for continuous time systems. For discrete time systems, the response is
+        evaluated around the unit circle such that
+
+             G(exp(j*omega*dt)) = mag*exp(j*phase).
+
+        Parameters
+        ----------
+        omega : array_like or float
+            A list, tuple, array, or scalar value of frequencies in
+            radians/sec at which the system will be evaluated.
+        squeeze: bool, optional (default=True)
+            If True and sys is single input, single output (SISO), return a
+            1D array or scalar depending on omega's length.
+
+        Returns
+        -------
+        mag : (self.outputs, self.inputs, len(omega)) or len(omega) ndarray
```

Comment:
```suggestion
        In general the system may be multiple input, multiple output (MIMO), where
        `m = self.inputs` number of inputs and `p = self.outputs` number of
        outputs.

        Parameters
        ----------
        s: float or array_like
            A list, tuple, array, or scalar value of frequencies in
            radians/sec at which the system will be evaluated.
        squeeze : bool, optional (default=True)
            If True and the system is single input single output (SISO), i.e. `m=1`,
            `p=1`, return a 1D array rather than a 3D array.

        Returns
        -------
        mag : (p, m, len(omega)) or (len(omega),) array 
            The magnitude (absolute value, not dB or log10) of the system
            frequency response. Array is ``(len(omega), )`` if
            and only if system is SISO and ``squeeze=True``.
        phase : (p, m, len(omega)) or (len(omega),) array
            The wrapped phase in radians of the system frequency response.
        omega : ndarray
            The (sorted) frequencies at which the response was evaluated.
            
```